### ISSUE TITLE
launch, and dev workflow after launch?

### ISSUE THREAD

**Issue by garrett-laroy-johnson** [parkerdavis1/CCAM#24]
Are we in a place where we can move this incredible website over to abstractconcrete.center ? We are sending off another grant app today and we would be well served by this gorg new site -- in case they take a peak. 

we can still do a more formal 

🚀 
L
A
U
N
C
H 
🔥 

once we have met again and are locked in on more design decisions.

About workflow post publication:

In thinking about the "fake resources" etc, there will continue to be things we will want to try and have each other look at (hopefully not for months and months, but you know what I mean). 

Is the next step to make a dev branch?

**Comment by parkerdavis1** [comment]
Yeah, I think generally the workflow ought to be pushing all changes to a Dev branch, which then gets built into a staging site, making sure things work, then merging those changes to the Main branch which gets built into the real site that the domain points to. 

Right now everything is connected to my various accounts (github, netlify, val.town (currently handling signup form)) which *at some point* should be switched to accounts under your own control, but not a priority if you’re trying to get the site **LIVE** today.

**Comment by parkerdavis1** [comment]
But yeah, copying things over to a dev branch and cleaning things up for presentability on the main branch to point the domain at, sounds good, go for it

**Comment by parkerdavis1** [comment]
Okay, there is a `dev` branch now, and that is what is being built and displayed at ccam-staging.netlify.app. 

Not sure how you would like to deploy the 'real' site, on another netlify instance that you could configure with your custom domain, or some other hosting service, whatever. Also we could transfer 'ownership' of the github repo over to @garrett-laroy-johnson – if I'm a collaborator it seems like I'd still be able to do all the things that I might need to do going forward.

**Comment by parkerdavis1** [comment]
@garrett-laroy-johnson @ioanbutiu all commits going forward should be pushed to the `dev` branch, not `main`.

**Comment by garrett-laroy-johnson** [comment]
main branch now points to www.abstractconcrete.center .

Next steps to be taken in about a month: 
1. purchase CCAM.world as our first official expenditure (hopefully in February).
2. set up CCAM.world as the main site. Set up a message on abstractconcrete.center that we are moving to ccam.world, and autoforward. Let abstractconcrete.center expire on May 3, 2024.

**Comment by garrett-laroy-johnson** [comment]
the www.ccam.world is ours!

a few questions:

@parkerdavis1 is it quick to prepare a forwarding page for abstractconcrete.center? 

question: if someone puts in some page that isn't the index at the root (e.g. https://abstractconcrete.center/resources/custom-json-for-oopsy-daisy/) 

will it still snag them and forward to ccam.world? 

And, correct me if I'm wrong, but I imagine it wouldn't forward them to https://ccam.world/resources/custom-json-for-oopsy-daisy/ 

not the biggest issue as I can't imagine there are a ton of links out there yet.

**Comment by parkerdavis1** [comment]
If you setup ccam.world with the same DNS settings as abstractconcrete.center, like the same changes you made to direct abstractconcrete.center towards the netlify site, I should be able to get that setup quickly. I will get back to you about specifics about redirects.

**Comment by parkerdavis1** [comment]
LAUNCH LAUNCH LAUNCH

I wanna get my name on that site. But I don't think I want the full picture/bio treatment (even though I have been a good boy) so I wonder how to slap it on there. I’d love to parlay this into my next $10,000+ contract.

**Comment by garrett-laroy-johnson** [comment]
You name it and you got it. I’ll give you a blurb too lol

**Comment by garrett-laroy-johnson** [comment]
# CCAM by Parker Davis 

?

just kidding. But yeah we can give you a credit on About? seriously though however you would like.

**Comment by parkerdavis1** [comment]
I'm making it so that `abstractconcrete.center/any/where/on/site` automatically/instantaneously redirects to `ccam.world/any/where/on/site`. 

Many people probably won't even notice that they were redirected to a new URL. This is either a good or a bad thing depending on your goals, ha. 

The other option is to make it so abstractconcrete.center will only show a small message about redirecting and redirect to the homepage ccam.world, not preserving any links

**Comment by parkerdavis1** [comment]
All abstractconcrete.center links now redirect to ccam.world links 🎉

**Comment by parkerdavis1** [comment]
Also, I added Ioan and I to the About page. 
<img width="1135" alt="Screenshot 2024-03-05 at 2 39 39 PM" src="https://github.com/parkerdavis1/CCAM/assets/92223355/697f5cf1-98b4-4ed3-8248-e129c9a4f6ad">
